src/run.ts

```diff
@@ -74,9 +74,9 @@ export default async function(helper: Helper, args: CreateWidgetArgs) {
 		name,
```

Comment:
@bitpshr We should be able to use `MockModules.ts`, to assert the arguments of `renderFiles` that will cover these changes.

I don't think we have mock modules in this repo, but you should be able to copy it from [@dojo/webpack-contrib](https://github.com/dojo/webpack-contrib/blob/master/tests/support/MockModule.ts) and then use it similar to the [build-time-render tests](https://github.com/dojo/webpack-contrib/blob/master/tests/unit/build-time-render/BuildTimeRender.ts#L27)